src/data/equiv/denumerable.lean

```diff
@@ -104,14 +129,19 @@ by simp; refl
 @[simp] theorem prod_nat_of_nat : of_nat (ℕ × ℕ) = unpair :=
 by funext; simp
 
+/-- `ℤ` is denumerable. -/
```

Comment:
is it worth putting these in? they're tautologous + I don't think instances need docstrings by linter

Reply:
I'm not sure. But it's not much effort to have them. You could argue that every docstring is tautological once you know enough.

Reply:
I think a docstring for `foo X` that says only "X is foo" is pointless. Can you expand these to say _what_ the equivalence is?

Reply:
```suggestion
/-- `ℤ` is denumerable via `equiv.int_equiv_nat`. -/
```
etc. But even that is perhaps not worth having, since doc-gen shows definitions too.

Reply:
I've now removed them. Not sure it's that bad to have tautological docstrings, but okay.